packages/cli/src/models/network/NetworkController.ts

```diff
@@ -428,17 +430,17 @@ export default class NetworkController {
     }
 
     log.info(`Publishing project to ${this.network}...`);
-    const simpleProject = <SimpleProject>(await this.fetchOrDeploy(this.currentVersion));
+    const proxyAdminProject = <ProxyAdminProject>(await this.fetchOrDeploy(this.currentVersion));
     const deployer = new AppProjectDeployer(this, this.packageVersion);
-    this.project = await deployer.fromSimpleProject(simpleProject);
+    this.project = await deployer.fromProxyAdminProject(proxyAdminProject);
```

Comment:
We need to make sure, when we implement the migrations, to first move from simple project to proxy admin project, if the user is coming from an older version.